main.go

```diff
@@ -169,26 +175,32 @@ func main() {
 	}
 }
 
-func prepareEventsToTrace(eventsToTrace []string, excludeEvents []string) ([]int32, error) {
+func prepareEventsToTrace(eventsToTrace []string, tagsToTrace []string, excludeEvents []string) ([]int32, error) {
 	var res []int32
 	eventsNameToID := make(map[string]int32, len(tracee.EventsIDToEvent))
 	for _, event := range tracee.EventsIDToEvent {
 		eventsNameToID[event.Name] = event.ID
 	}
-	if eventsToTrace == nil {
+	tagsToEvents := make(map[string][]int32)
+	for id, event := range tracee.EventsIDToEvent {
```

Comment:
you can combine this loop with the one that appears 5 lines before

Reply:
Right, thanks